pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -1773,6 +1833,25 @@ jobs:
     get_params:
       skip_download: true
 
+# Run DB Teardown
+- name: "CI Reset DB"
+  on_failure: *slack_failure_alert_ci
+  on_error: *slack_error_alert_ci
+  plan:
+  - get: every-midnight
+    trigger: true
+  - get: census-rm-kubernetes-repo
```

Comment:
Could use existing census-rm-kubernetes-repo instead

Reply:
Changed to `census-rm-kubernetes-optional-repo` instead